test/models/recipe_ingredient_test.rb

```diff
@@ -1,8 +1,8 @@
 require "test_helper"
 
 class RecipeIngredientTest < ActiveSupport::TestCase
-  test "should have a recipe id, ingredient id, unity id and quantity id" do
-    r_i = recipe_ingredients(:one)
+  test "should have a recipe id, ingredient id, a unity and a quantity" do
+    r_i = recipe_ingredients(:ri_pain)
     assert r_i.valid?
```

Comment:
```suggestion
    recipe_ingredient = recipe_ingredients(:ri_pain)
    assert recipe_ingredient.valid?
```
Ideally `ri_pain` should be named `recipe_ingredient_bread` (using bread for language consistency with the code).